index.html

```diff
@@ -20,157 +20,162 @@
-            <p>
-              Providing donated bikes and accessories to refugees and asylum
-              seekers in Scotland.
-            </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
-          </div>
+        <div>
+          <h2 class="heading-underline">Learn more</h2>
 
-          <div>
-            <h2 class="heading-underline">Learn more</h2>
-
-            <div>
-              <div class="article">
-                <h3 class="article__title">Why do refugees need bikes?</h3>
-                <p class="article__summary">
-                  Many refugees are placed in housing in areas where there are
-                  few jobs. Bikes provide low-cost transportation so that they
-                  can get to work, manage child care and do their shopping.
-                </p>
-                <p class="article__read-more">
-                  <a href="#">
-                    <span class="arrow-right"></span>
-                    Learn more
-                  </a>
-                </p>
-              </div>
-
-              <div class="article">
-                <h3 class="article__title">How can I help?</h3>
-                <p class="article__summary">
-                  We need lots of bikes and bike accessories! If you have an
-                  old bike you don't use, bring it to one of our dropoff
-                  events. Or come attend one of our fundraisers.
-                </p>
-                <p class="article__read-more">
-                  <a href="#">
-                    <span class="arrow-right"></span>
-                    Learn more
-                  </a>
-                </p>
-              </div>
+          <div class="article-container">
+            <div class="article">
+              <h3 class="article__title">Why do refugees need bikes?</h3>
+              <p class="article__summary">
+                Many refugees are placed in housing in areas where there are few
+                jobs. Bikes provide low-cost transportation so that they can get
+                to work, manage child care and do their shopping.
+              </p>
+              <p class="article__read-more">
+                <a href="#">
+                  <span class="arrow-right"></span>
+                  Learn more
+                </a>
+              </p>
             </div>
-          </div>
-        </div>
 
-        <div class="sidebar">
-          <h2 class="heading-underline">Upcoming events</h2>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="spring-fundraisers_thumbnail.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Join us for our Spring fundraisers</a>
-              </h3>
+            <div class="article">
+              <h3 class="article__title">How can I help?</h3>
```

Comment:
A stretch goal - what is an alternative way of styling the **nested** tags? Do we really need to apply an explicit style to h3 here or there is another way of handling this in CSS?